py3status/modules/battery_level.py

```diff
@@ -218,8 +218,7 @@ def _extract_battery_information_from_acpi(self):
         Battery 0: design capacity 5566 mAh, last full capacity 5156 mAh = 92%
         Battery 1: Unknown, 98%
         Battery 1: design capacity 1879 mAh, last full capacity 1370 mAh = 72%
-        '''
-
```

Comment:
I don't like this line being removed but I'll merge it anyhow.  Blank lines are important don't murder them all.

They often help with readability